index.html

```diff
@@ -25,10 +25,10 @@ <h1>I'm Eugene.<br>
                 <h3>LET'S CONNECT</h3>
                 <ul class="contact">
                     <li><img class="icons" src="Vector.svg"></li>
-                    <li><img class="icons" src="Linkedin icon.svg"></li>
-                    <li><img class="icons" src="Vector (1).svg" alt=""></li>
-                    <li><img class="icons" src="Vector (2).svg" alt=""></li>
-                    <li><img class="icons" src="Vector (3).svg" alt=""></li>
+                    <li><img class="icons" src="linkedin.svg"></li>
+                    <li><img class="icons" src="vector1.svg" alt=""></li>
+                    <li><img class="icons" src="vector2.svg" alt=""></li>
+                    <li><img class="icons" src="vector3.svg" alt=""></li>
```

Comment:
- [ ] As suggested by the previous reviewer, Kindly make sure to wrap each of the social media icons in an `a` tag as this will help you to include your social media profile links in the `href` which will help your users to follow and connect with you. See below for more detail:
```
<ul>
  <li><a href="#">social media icon</a></li>
  <li><a href="#">social media icon</a></li>
  <li><a href="#">social media icon</a></li>
  <li><a href="#">social media icon</a></li>
</ul>
```